base/applications/mspaint/winproc.cpp

```diff
@@ -263,6 +263,8 @@ LRESULT CMainWindow::OnMouseWheel(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL&
 
 LRESULT CMainWindow::OnDropFiles(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
 {
+    CWaitCursor waitCursor;
```

Comment:
Weird to get a wait cursor, while at the same time the drag operation may change the cursor to the drag one.
Also, what about the cursor when you get the save-dialog confirmation being shown?
If the cursor stays in "wait" mode, it may actually be better to set it in wait mode only in the actual image-loading function?

Reply:
Ok, deleted.

Reply:
The dialog box might change the set'ed cursor.